contrib/cirrus/runner.sh

```diff
@@ -55,6 +55,7 @@ function _run_unit() {
 }
 
 function _run_apiv2() {
+    warn "Testing podman API with podman client"
```

Comment:
Even looking at these additions right next to each other, my eye completely missed the difference. Could you all-caps PODMAN and DOCKER, perhaps even asteriskize them, to make it super-duper clear to someone looking at logs?

Reply:
actually that's a good idea...